src/components/Select/index.test.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Select from "./index";
+
+describe("Select", () => {
+  it("renders the default option when no defaultValue is set", () => {
```

Comment:
This was a bit confusing for me with the use of "default option", I think what you're saying is:

```suggestion
  it("renders the prompt as the default option when no defaultValue is set", () => {
```